### DISCUSSION TITLE
Divergence in combustion behavior between FDS 6.8 & 6.9 - Boundary Fuel Model

### DISCUSSION THREAD

**Discussion by sebastianmezaurotadze** [firemodels/fds#13237] [Q&A] [answered]
I'm running some simulations to validate wood combustion behavior. My simulation runs on FDS 6.8 always seem to fail from Numerical Instabilities caused by spiking release of plasma. This issue was resolved once I switched to 6.9, but I'm concerned on the stark difference of how the physics behaves between versions since I'm still running the same FDS file. 

So I'm wondering, are there any big physics modeling updates or bug fixes on 6.8 related to numerical instabilities that were rolled out on 6.9?

Might be useful to note that I'm running v6.8 / v6.9 by running FDS on a cloud service through pyrosim.

**Comment by ericvmueller** [comment, marked as answer]
There were a few changes, particularly around removing `AIR` as an implicit material within the solid ...which had an effect on how the surface would regress as burning and on the approximation of convective heat transfer. A bug with implementing the `SURFACE_OXIDATION_MODEL` was also fixed, if you are using that
https://github.com/firemodels/fds/pull/11813
https://github.com/firemodels/fds/pull/11857
https://github.com/firemodels/fds/pull/11791

In general, this method has been a bit fragile, especially if you try to include char oxidation, but the changes have tried to get away from setting a `MINIMUM_BURNOUT_TIME` which has been a fallback for improving stability. I would consider this a research feature and we are working to improve consistency between this and other methods for predicting burning rates and flame spread.
